functions/Invoke-DbaAdvancedUpdate.ps1

```diff
@@ -53,7 +56,8 @@ Function Invoke-DbaAdvancedUpdate {
         [ValidateSet('Default', 'Basic', 'Negotiate', 'NegotiateWithImplicitCredential', 'Credssp', 'Digest', 'Kerberos')]
         [string]$Authentication = 'Credssp',
         [pscredential]$Credential,
-        [switch]$EnableException
+        [switch]$EnableException,
+        [string]$ExtractPath
```

Comment:
Ensure that `EnableException` is always the last parameter for usability purposes (tabbing). Same for CBH, it should be before Confirm in this case.

Reply:
Ok, got it. CBH?

Reply:
I moved it, not sure about the CBH bit.